examples/6-advanced-flows.html

```diff
@@ -0,0 +1,213 @@
+                "answer": "intro-context"
+            }
+        ],
+    },
+    "chapter-two": {
+        "says": [
+            "Very good. Have you ...",
+            "[Question to filter out respondents | Create a branch in the logic]",
+        ],
+        "reply": [
+            {
+                "question": "I have.",
+                "answer": "chapter-three"
+            },
+            {
+                "question": "I have not. [Escape Valve hit | End]",
+                "answer": "end" // An example of using the Escape Valve for ineligible, Not Applicable participants 
+            },
+        ]
+    },
+    "chapter-three": {
+        "says": [
+            "Go ahead to the next chapter.",
+            "<img src=http://meanbusiness.com/wp-content/uploads/2018/03/Coral-Participate-Phone_and_Laptop.jpg />",
+        ],
+        "reply": [
+            {
+                "question": "OK, I will.", // A Gimme: An easy answer with fewer choices to keep the egagement going. Useful during longer dialogues
+                "answer": "chapter-four"
+            }
+        ]
+    },
+    "chapter-four": {
+        "says": [
+            "Thank you very much.",
+            "<img src=http://meanbusiness.com/wp-content/uploads/2018/02/mel_b-Thank-You.gif />",
+            "More? Or Quit while we're behind?",
+        ],
+        "reply": [
+            {
+                "question": "Quit [Escape Valve hit]",
+                "answer": "end"
+            },
+            {
+                "question": "More",
+                "answer": "chapter-five"
+            },
+        ],
+    },
+    "chapter-five": {
+        "says": [
+            "[Answering 'Yes' here will open a new window with a page from another site]", // call a function to an external resource / application outside of chat-bubble's purview
+        ],
+        "reply": [
+            {
+                "question": "Not interested, thanks. [Escape Valve hit]",
+                "answer": "end"
+            },
+            {
+                "question": "Yes",
+                "answer": "externalResourceFunction"
+            },
+        ]
+    },
+    "end": {
+        "says": [
+            "[Escape Valve] Thanks for your time & attention.",
+            "<img src=https://media.giphy.com/media/xUOxeXsWhw6DCW1cSA/giphy.gif>",
+            "[Other messaging to encourage starting over or coming back again]"
+        ],
+        "reply": [
+            {
+                "question": "[Start Over] 😁",
+                "answer": "ice"
+            },
+        ]
+    },
+    // another conversation object that can be queued from within
+    // any other conversation object, including itself
+} // end conversation object
+)
+
+    // this function is called when user asks for banana
+    externalResourceFunction = function () {
+        window.open("https://meanbusiness.com/gamepath/coralproject/coral-swiper-180306/","_blank",
```

Comment:
What is this website?

Reply:
[As below](https://github.com/dmitrizzle/chat-bubble/pull/53#discussion_r179615446), this is another example use case I did for the Coral Project to represent personality types in News communities. Will update this link.